src/scss/cs/homepage.scss

```diff
@@ -125,3 +125,170 @@
     border-color: $highlight-navy;
   }
 }
+
+// ----------------------------------------------
+// Homepage Resources Section
+// ----------------------------------------------
+
+.resources-section {
```

Comment:
nice. small tip to make it even better, something like this could work: 

```scss
.resources {
  &-section {
    background-colour: ....
  }

  &-background {
    background-colour: ....
  }

  &-sub-heading-container {
    display: flex;
  }
}

```

applying that to the whole new set of css would really make best use of scss here :)

Reply:
BEM _ish_. Not fully following it, but not far off :) It's a good analogy to follow.